docs/topics/basics/keyspaces/README.md

```diff
@@ -23,13 +25,32 @@ CassFuture* use_future
 /* Check future result... */
 ```
 
-## Single session mulitple keyspaces
+## Single Session and Mulitple Keyspaces
 
-It is possible to use multiple keyspaces from a single session by full qualifying the table names in your queries e.g. `keyspace_name.table_name`.
+It is possible to interact with multiple keyspaces using a single session object by fully qualifying the table names in your queries e.g. `keyspace_name.table_name`.
 
-Examples:
+### Examples
 
-```cql
+```sql
 SELECT * FROM keyspace_name.table_name WHERE ...;
 INSERT INTO keyspace_name.table_name (...) VALUES (...);
 ```
+
+## Creating Keyspaces and Tables
+
+It is also possible to create keyspaces and tables by executing CQL using a session object.
+
+### Examples
+
+```sql
```

Comment:
When I ran the generated docs through the browser, the cql markdown didn't show up.  Either way I can change it to cql for future markdown adoption.